### PR TITLE
fix: Update f-5 hugo theme version

### PR DESCRIPTION
**Pull request by ashutoshpradhan-f5**
### Proposed changes
Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue here in this description (not in the title of the PR).

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by ashutoshpradhan-f5** [comment]
@lucacome - Pls review and approve. Do you want to merge it to Master as no content update has been done for next release to Master yet? CC: @pleshakov

**Review by ashutoshpradhan-f5 — COMMENTED** [review]
Updated f5-hugo theme version

File: go.mod [hunk 1]
```diff
@@ -17,6 +17,7 @@ require (
 	github.com/prometheus/client_golang v1.11.0
 	github.com/spiffe/go-spiffe v1.1.0
 	github.com/stretchr/testify v1.7.0 // indirect
+	gitlab.com/f5/nginx/controller/poc/f5-hugo v0.11.5 // indirect
```
Comment (lucacome) [inline go.mod:20]: why is this needed now? shouldn't it be only in `docs/go.mod`?

Comment (ashutoshpradhan-f5) [reply go.mod:20]: @jputrino

Comment (ashutoshpradhan-f5) [reply go.mod:20]: Closing this PR, creating another one to make changes only to Docs folder.

**Pull request closed without merging**